serverless/virus-scanner/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:16.20-bullseye-slim
+
+# Install aws-lambda-cpp build dependencies
+RUN apt-get update && \
+  apt-get install -y \
+  g++ \
+  make \
+  cmake \
+  unzip \
+  libcurl4-openssl-dev \
+  clamav \
+  clamav-daemon \
+  autoconf \
+  libtool \
+  automake \ 
+  python3
+
+WORKDIR /function
+
+# install app dependencies
+COPY package.json package-lock.json ./
+RUN npm ci
+
+# Point clamav log and socket to /tmp folder
+RUN sed -i 's/LogFile \/var\/log\/clamav\/clamav.log/LogFile \/tmp\/clamav.log/g' /etc/clamav/clamd.conf
+RUN sed -i 's/LocalSocket \/var\/run\/clamav\/clamd.ctl/LocalSocket \/tmp\/clamd.ctl/g' /etc/clamav/clamd.conf
+
+# !! Below comment was taken from care360 repo where the virus scanner was forked from!!
+# This is a hack to give the default lambda user permissions write access to the Local UNIX socketfile and logfile
+# The appropriate GID (990) and User (sbx_user1051) to add was found out through logging the user info at lambda runtime
+# GID and User is assumed to be deterministic and hence can be hardcoded
+# An alternative would be to get the current user + group details at runtime at the app level code, then run clamd
+# TODO (Austin): If the function breaks in the future because GID / User is non-deterministic, experiment with executing command at app runtime
```

Comment:
Thanks! actually i don't think this is really a TODO, it's more of an info note in case something breaks (`If the function breaks...`). So i've removed the todo here

[d8eee93](https://github.com/opengovsg/FormSG/pull/6642/commits/d8eee931805e77272c69600afdf71eef7dc2fba4)